met/src/basic/vx_util/CircularTemplate.cc

```diff
@@ -56,126 +34,92 @@
-  // center of the circle.
-  int maxOffset = static_cast<int>(floor(radius));
-  if (evenWidth){
-	  maxOffset++;
-  }
+   // Need to increase the area we look at if the width is even, because
+   // some valid offset points will actually be farther from the reference point
+   // than the radius, because the reference point is offset from the true
+   // center of the circle.
+
+   int maxOffset = static_cast<int>(floor(radius));
+   if(evenWidth) maxOffset++;
   
-  int minOffset = static_cast<int>(floor(-1 * radius));
+   int minOffset = static_cast<int>(floor(-1 * radius));
 	  
-  for (int y = minOffset; y <= maxOffset; y++)
-  {
-    for (int x = minOffset; x <= maxOffset; x++)
-    {
-	    double double_x = (double)x;
-	    double double_y = (double)y;
-
-	    if (evenWidth){
-		    // if width is even, the reference point is actually shifted 1/2 a grid spacing down and to the left,
-		    // from the true center of the circle.
-		    // 
-		    // so when we calculate distance, we need to subtract .5 so that the distance reflects the distance from the center
-		    // of the circle, instead of the distance from the reference.
-		    //
-		    // for example - a circle with width == 4.  The reference point is the lower left corner of the center square.
-		    // the point directly below that is at (0,-1), but it's actually (-.5, -1.5) from the center of the circle.
-		    //
-		    // another example - same circle.  The point directly to the right of the reference point is (1,0), but it's
-		    // actually (.5,-.5) from the center.
+   for(int y = minOffset; y <= maxOffset; y++) {
+      for(int x = minOffset; x <= maxOffset; x++) {
+         double double_x = (double)x;
+         double double_y = (double)y;
+
+	      if(evenWidth) {
+            // if width is even, the reference point is actually shifted 1/2 a grid spacing down and to the left,
+            // from the true center of the circle.
+		      //
+		      // so when we calculate distance, we need to subtract .5 so that the distance reflects the distance from the center
+		      // of the circle, instead of the distance from the reference.
+		      //
+		      // for example - a circle with width == 4.  The reference point is the lower left corner of the center square.
+		      // the point directly below that is at (0,-1), but it's actually (-.5, -1.5) from the center of the circle.
+		      //
+		      // another example - same circle.  The point directly to the right of the reference point is (1,0), but it's
+		      // actually (.5,-.5) from the center.
 		   
-		    double_x -= 0.5;
-		    double_y -= 0.5;		   
-	    }
-	    double distance= sqrt((double_x * double_x) + (double_y * double_y));
-
-	    if (distance <= radius)
-		    {
-			    _addOffset(x, y);
-		    }
-      
-    } /* endfor - x */
-    
-  } /* endfor - y */
-
-  _setEdgeOffsets();
- 
-}
+            double_x -= 0.5;
+		      double_y -= 0.5;
```

Comment:
The indentation here and in the comment block above seem to have been messed up somehow. This is obviously not crucial to fix but it would be nice for consistent formatting.

Reply:
Thanks for catching this. It's annoying spaces vs tabs, and I missed. Should be [fixed](https://github.com/dtcenter/MET/commit/39b882f60d50ce1dafdb317a9a29b4e9a1312a5e) now.